include/Options/Business/Sync.php

```diff
@@ -43,6 +43,7 @@ protected function get_data_structure(): array {
 		add_filter( 'lang_dir_for_domain', '__return_false' ); // Prevents loading the translations too early.
 		$enum = array_keys( \PLL_Settings_Sync::list_metas_to_sync() );
 		remove_filter( 'lang_dir_for_domain', '__return_false' );
+		unload_textdomain( 'polylang' ); // To ensure string translation later in the process will be done.
```

Comment:
1. I'm surprised that it works like this without allowing texdomain reload.  
2. "string translation" is confusing (as we use this expression for one of our feature), so I suggest a new comment:
3. Is it possible to write a test?
```suggestion
		unload_textdomain( 'polylang', true ); // Required to allow `_load_textdomain_just_in_time()` to load the translations.
```

Reply:
I confirm that the 2nd parameter isn't needed to make `_load_textdomain_just_in_time()` reload the textdomain.
In this case
See https://github.com/WordPress/WordPress/blob/6.7.1/wp-includes/l10n.php#L921
Only one locale is concerned at this step.

I'm looking to write a test.